recipe/meta.yaml

```
@@ -25,45 +24,43 @@ requirements:
     - {{ compiler('cxx') }}
     - pkg-config
     - m2w64-toolchain  # [win]
-    - patch            # [unix]
-    - m2-patch         # [win]
+    - patch            # [s390x or aarch64]
   host:
-    - setuptools >=18.0
-    - m2w64-pkg-config  # [win]
+    - setuptools
+    # - m2w64-pkg-config 0.29.1  # [win]
     - python
     - pip
-    - numpy-devel {{ numpy }}  # [not (osx and arm64) and py<=39]
-    - numpy {{ numpy }}        # [(osx and arm64) or py>39]
-    - cython
-    - zlib
-    - bzip2
-    - xz
-    - zstd
-    - lz4-c                 # [not win]
-    - lz4-c >=1.9.3         # [win]
-    - libpng
-    - libwebp
-    - libwebp-base
-    - jxrlib
-    - jpeg
-    - giflib
-    - openjpeg
-    - blosc
-    - lcms2  # [not win32]
-    - libaec
-    - brotli
-    - libzopfli
-    - charls          # [not s390x]
-    - snappy
+    - numpy 1.24
```

```suggestion
    - numpy {{ numpy }}
```
Like explained, we have to make sure that the numpy version we use to compile is consistent for a given python version (at least in our ecosystem) because of ABI compatibility.